### DISCUSSION TITLE
Issues with Content Delivery when README.md contains hundreds of images

### DISCUSSION THREAD

**Discussion by ace-dent** [community/community#56354] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

My project has a README.md with a gallery section.
The gallery has a 200-300 small images (~100 bytes each).
When viewing on a desktop browser, only half the images load (more load with each page refresh).
When viewing on a mobile device over cellular data, many images don't load or the whole page is just blocked.
I believe that GitHub's CDN is throttling or blocking the many transfer requests...?

Ideally we would embed small images directly in the markdown as html `<img>` with base64 encoded `src`. BUT GitHub's markdown parser strips out such encoded data. :-(
It also seems other methods are blocked, e.g.: css-sprites; also `loading="lazy"` gets stripped, etc....

Would love an explanation and/or solution 🙏 

https://github.com/ace-dent/8x8.me/blob/cb6b07d281b2271dc472cf72c3b1c4e0fabb5798/README.md

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ace-dent ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
